### ISSUE TITLE
initramfs

### ISSUE THREAD

**Issue by psinagra** [pieroproietti/penguins-eggs-legacy#125]
Hello! Thank you again for your work on eggs! As I've said before, it's an amazing tool. I just wanted to bring something to your attention. After the installation of the new system using calamares and the default eggs theme on eggs 9.1.17, I tried to update initramfs. It gave me the message in the attached picture.
<img width="604" alt="Screen Shot 2022-05-12 at 3 11 14 PM" src="https://user-images.githubusercontent.com/23179142/168152106-2cbb4e56-4966-429c-9f60-f4282cab7580.png">
 
The system I work with and use is Ubuntu Jammy 22.04 with Cinnamon Desktop Environment.
Is there a work around for this? On my original system, I was able to update it just fine. But after installing the system I get the message in the picture when I try to update initramfs. Thank you in advance as always!

**Comment by pieroproietti** [comment]
I have the same problem, 

don't understood well why it happen, but seem to not create problems. 

If someone come with same suggestions we can try to solve it.

**Comment by psinagra** [comment]
As far as running the system, you're right it doesn't cause any problems. But it may present an issue when updating the kernel in the future.

**Comment by pieroproietti** [comment]
we can try to ìnvestigate a bit...

https://askubuntu.com/questions/1116778/how-to-set-the-resume-variable-to-override-these-issues

```
printf "RESUME=UUID=$(blkid | awk -F\" '/swap/ {print $2}')\n" | sudo tee /etc/initramfs-tools/conf.d/resume
```
of course this work on installed systems. On the live, probably we can remove /etc/initramfs-tools/conf.d/resume.

Look if you can found exactly that is the problem and we can remove it.

**Comment by psinagra** [comment]
I will definitely try this. I'll do it on the original system to create the file, and then I'll use eggs to create the iso. I'll let you know if it works. Thanks again!

**Comment by psinagra** [comment]
Question: Does this also happen in Debian? Also, on my original Ubuntu Jammy system, there is no resume file at all.

**Comment by pieroproietti** [comment]
Yes, as I remember. Today, after we speach I included /etc/initramfs-tools/conf/resume in the exclude list, you can check the last commit. I made same versions starting with ubuntu jammy naked (without gui) and don't saw this problem.

**Comment by psinagra** [comment]
Thank you so much. I really appreciate the help! I'll try it out again.

**Comment by psinagra** [comment]
Tried the latest version. Still gives the same message in the picture above.

**Comment by pieroproietti** [comment]
I'm preparing an updated version of naked ubuntu jammy, after I will "wear" it with cinnamon with the command ```eggs warddrobe get``` and ```sudo eggs wardrobe wear duck```. This will install a cinnamon configuration I made using eggs wardrobe and will see that happen.

**Comment by pieroproietti** [comment]
Just finished, I don't have the issue!
I created an ubuntu with cinnamon, installed calamares with ```sudo eggs calamares --install```, created iso with ```sudo eggs produce --release```.

Using: eggs version 9.1.24 on ubuntu 22.04

You can find both the isos and the package on [penguins-eggs sourceforge page](https://sourceforge.net/projects/penguins-eggs/files/iso/ubuntu/jammy/)
* egg-of-ubuntu-jammy-nake... (the CLI iso)
* egg-of-ubuntu-jammy-duck... (cinnamon iso)

I hope this can help.

**Comment by psinagra** [comment]
I tried the new version of eggs 9.1.24. I installed it on my system, and proceeded to produce an egg. I installed the new system and as soon as I updated initramfs, I got the same message. 

I will now try the iso for egg-of-ubuntu-jammy-naked and see what happens...

**Comment by pieroproietti** [comment]
To investigate - btw I have differents problems in my cinnamon version: gnome-terminal don't start and I installed xfce4-terminal to work - I installed it with calamares and give ```update-initramfs -u```

This is the result:

```
I: The initramfs will attempt to resume from /dev/sda2
I: (UUID=433e9bfd-4c0d-4206-9023-2d6105cd1ec4)
I: Set the RESUME variable to override this.
live-boot: core filesystems devices utils udev blockdev iscsi dns.
artisan@duck:~$ sudo blkid 
/dev/sr0: BLOCK_SIZE="2048" UUID="2022-05-25-07-41-15-00" LABEL="duck" TYPE="iso9660" PTUUID="62d864cc" PTTYPE="dos"
/dev/sda2: LABEL="swap" UUID="433e9bfd-4c0d-4206-9023-2d6105cd1ec4" TYPE="swap" PARTUUID="6daaa46d-02"
/dev/sda1: UUID="0ec8ccec-81a8-48b1-9f20-1b43ec4cdc8d" BLOCK_SIZE="4096" TYPE="ext4" PARTUUID="6daaa46d-01"
```
In my eyes this is correct: it will resume from UUID="433e9bfd-4c0d-4206-9023-2d6105cd1ec4"  who is my swap partition.

But this experience put me in the idea to remove eggs and dependencies using calamares during installation. 

Until now I would not remove eggs altogether in the hope that it would be useful for a rebuild of the installed version. However, this leads to a number of problems, not the least of which is the necessary presence of the ``live-*`` packages. 

I plan to change this behavior with the next version, in which producing an iso with the ``--release`` option will also remove eggs and live* packages.

While the system loses the ability to reproduce itself, the operation is easily reversible by re-installing eggs.

**Comment by pieroproietti** [comment]
OK, I released eggs-9.1.25. From the changelog:
* produce: 
  * flag --release now configure calamares to remove calamares. eggs and live* dependencies
  * links on desktop, bugfix
* calamares: renamed option --final to the more appropriate --release

**Comment by psinagra** [comment]
Ok. Let me download it now and give it a try!
